tools/datamash/datamash-ops.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0"?>
+<tool id="DatamashOps" name="Datamash" version="1.0.6">
+    <description>(operations on tabular data)</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <expand macro="stdio" />
+    <command>
+        <![CDATA[
+            datamash
+                $header_in
+                $header_out
+                $need_sort
+                $print_full_line
+                $ignore_case
+                #if str($grouping).strip()
+                --group '$grouping'
+                #end if
+                #for $oper in $operations
+                    ${oper.op_name}
+                    ${oper.op_column}
+                #end for
+                < $in_file > $out_file
+          ]]>
+    </command>
+    <expand macro="inputs_outputs">
+        <param help="Example: 1,4 - To group by the first and fourth fields. Leave empty to perform operation on entire file as one group." label="Group by fields" name="grouping" type="text">
```

Comment:
`Example: 1,4 - To group by the first and fourth fields.`

I'd write this as:

`Example: to group by the first and fourth fields, use 1,4 .`